### PR TITLE
Fix powerpacks double reloading

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Fix powerpacks double reloading

Closes: https://github.com/tgstation/TerraGov-Marine-Corps/issues/15729
## Why It's Good For The Game
bugfix
## Changelog
:cl:
fix: Fix powerpacks double reloading
/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
Does this fix the ICC ammo belt and flamethrowers back fuel?

**Comment by Xander3359** [comment]
> Does this fix the ICC ammo belt and flamethrowers back fuel?

Anything that is a worn magazine

File: code/game/objects/items/power_cells.dm [hunk 1]
```diff
@@ -23,6 +23,8 @@
 	var/charge_delay = 0 // How long it takes for the cell to start recharging after last use
 	///used to track what set of overlays to use to display charge level
 	var/charge_overlay = "cell"
+	///flags specifically for magazines.
+	var/magazine_flags
```
Comment (Lumipharon) [inline code/game/objects/items/power_cells.dm:27]: ?why

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/game/objects/items/power_cells.dm (hunk 1 above)
Comment (Xander3359) [reply code/game/objects/items/power_cells.dm:27]: I forgor

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**